src/Nethermind/Nethermind.Synchronization/ParallelSync/SyncFeed.cs

```diff
@@ -44,7 +44,6 @@ private void ChangeState(SyncFeedState newState)
         public virtual void Finish()
         {
             ChangeState(SyncFeedState.Finished);
-            GC.Collect(2, GCCollectionMode.Aggressive, true, true);
```

Comment:
I think this GC was also fired on Headers, Blocks and Receipts!

Reply:
Thats fine, the state take most of the allocation.

Reply:
ok